contracts/test/tests/TestACLInterpreter.sol

```diff
@@ -87,9 +87,13 @@ contract TestACLInterpreter is ACL, ACLHelper {
 
         // conditional oracle returns true if first param > 0
         ConditionalOracle conditionalOracle = new ConditionalOracle();
-
         assertEval(arr(uint256(1)), ORACLE_PARAM_ID, Op.EQ, uint256(conditionalOracle), true);
         assertEval(arr(uint256(0), uint256(1)), ORACLE_PARAM_ID, Op.EQ, uint256(conditionalOracle), false);
+
+        // only owner oracle returns true if sender is the defined owner
+        OnlyOwnerOracle onlyOwnerOracle = new OnlyOwnerOracle(msg.sender);
```

Comment:
Might be worth adding another "complex" example with just a bit of logic and the `OnlyOwnerOracle()` to make sure we're threading through the `user` correctly.

Reply:
Added! Let me know!